### PR TITLE
add that you need to have done an ad-hoc apk upload before supply init

### PR DESCRIPTION
**Pull request by asterixorobelix**

### CODE REVIEW

**Comment by janpio** [comment]
Very good point.

I thought we already had mentioned that somewhere though... maybe only on the `supply` page itself and not on the setup?

**Comment by asterixorobelix** [comment]
@janpio this is documentation, no need to be DRY. In my opinion reiteration is good in documentation. this point was missed by me when I was reading the docs

File: docs/getting-started/android/setup.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ The most interesting file is `fastlane/Fastfile`, which contains all the informa
 
 _supply_ is a _fastlane_ tool that uploads app metadata, screenshots and binaries to Google Play. You can also select tracks for builds and promote builds to production!
 
+For _supply_ to be able to initialize, you need to have successfully uploaded an apk to you Google Play Console app at least once.
```
Comment (janpio) [inline docs/getting-started/android/setup.md]:
```suggestion
For _supply_ to be able to initialize, you need to have successfully uploaded an APK to your app in the Google Play Console at least once.
```

**Comment by janpio** [comment]
That wasn't a criticism, I was just trying to get the full picture.  
If we have similar wording somewhere else, we should make sure it is consistent.

File: docs/getting-started/android/setup.md (hunk 1 above)
Comment (rogerluan) [reply docs/getting-started/android/setup.md:36]:
```suggestion
For _supply_ to be able to initialize, you need to have successfully uploaded an APK to your app in the Google Play Console at least once.
```

**Review by rogerluan — CHANGES_REQUESTED** [review]

**Comment by rogerluan** [comment]
Closing in favor of https://github.com/fastlane/docs/pull/1030

**Pull request closed without merging**